hooks/node_handlers/base_render_handler.py

```diff
@@ -0,0 +1,374 @@
+
+    def _populate_aov_names(self, node, parent_parm_name, src_parm_name, dest_parm_name):
+        """
+        Populate sgtk aov names from the original aov names.
+
+        :param node: A :class:`hou.Node` instance.
+        :param str parent_parm_name: The parent parm name.
+        :param str src_parm_name: The source parm name.
+        :param str dest_parm_name: The sgtk parm name.
+        """
+        parent_parm = node.parm(parent_parm_name)
+        count = parent_parm.eval() + 1
+        for index in range(1, count):
+            src_parm = node.parm(src_parm_name.format(index))
+            channel_name = src_parm.evalAsString()
+            dest_parm = node.parm(dest_parm_name.format(index))
+            dest_parm.set(channel_name)
+
+    def _populate_from_fields(self, node, fields):
+        """
+        Populate the node from template fields.
+
+        :param node: A :class:`hou.Node` instance.
+        :param dict fields: The template fields.
+        """
+        super(BaseRenderNodeHandler, self)._populate_from_fields(node, fields)
+        sgtk_pass_name = node.parm(self.SGTK_PASS_NAME)
+        sgtk_pass_name.set(fields.get("identifier", ""))
+        self._populate_aov_names(
+            node, 
+            self.AOV_COUNT,
+            self.AOV_NAME_TMPL,
+            self.SGTK_AOV_NAME_TMPL
+        )
+
+    def _get_multi_parm_output_paths_and_templates(
+            self,
+            node,
+            count_parm_name,
+            use_file_parm_tmpl,
+            file_parm_tmpl,
+            work_template_name,
+            publish_template_name,
+            paths_and_templates
+        ):
+        """
+        Get the output path and the templates used for the given multi parms.
+
+        :param node: A :class:`hou.Node` instance.
+        :param str count_parm_name: The name of the count parameter to query.
+        :param str use_file_parm_tmpl: The string template for the use file parm name.
+        :param str file_parm_tmpl: The string template for the file parm name.
+        :param work_template: The work :class:`sgtk.Template` for this parm.
+        :param publish_template: The publish :class:`sgtk.Template` for this parm.
+        :param list paths_and_templates: The current list of paths and templates
+            to append to.
+        """
+        aov_work_template = self._get_template(work_template_name)
+        aov_publish_template = self._get_template(publish_template_name)
+        aov_count = node.parm(count_parm_name)
+        count = aov_count.eval() + 1
+        for index in range(1, count):
+            aov_enabled = node.parm(use_file_parm_tmpl.format(index))
+            if not aov_enabled.eval():
+                continue
+            self._get_output_path_and_templates_for_parm(
+                node,
+                file_parm_tmpl.format(index),
+                aov_work_template,
+                aov_publish_template,
+                paths_and_templates
+            )
```

Comment:
Could simplify and remove extra `continue`

```suggestion
            if aov_enabled.eval():
                self._get_output_path_and_templates_for_parm(
                    node,
                    file_parm_tmpl.format(index),
                    aov_work_template,
                    aov_publish_template,
                    paths_and_templates
                )
```